src/App.tsx

```diff
@@ -1,109 +1,77 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const totalItems: number = 42;
+  const values = [3, 5, 10, 20];
+
+  const [numItems, setNumItems] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const [from, setFrom] = useState(1);
+  const [to, setTo] = useState(numItems);
+
+  useEffect(() => {
+    let newFrom = 1;
+    let newTo = numItems;
+
+    if (currentPage) {
+      newFrom = (currentPage - 1) * numItems + 1;
+      newTo = Math.min(currentPage * numItems, totalItems);
```

Comment:
The check for `currentPage` is redundant here since `currentPage` is initialized with a value and has no way of being set to a falsy value. You can safely remove this check.